paseos/paseos.py

```diff
@@ -27,6 +27,9 @@ class PASEOS:
     # The actor of the device this is running on
     _local_actor = None
 
+    # List of communication links
+    _communication_links = None
```

Comment:
as mentioned, I would suggest an actor is responsible to maintain its own comm links. At the moment, if you want to run actors on different devices (not sharing memory , using e.g. MPI) you can simply exchange actors between devices. Having this here is very prone to error, since you could forget to manually also share the comm links at it is the only thing not contained within the actor